src/main/java/org/broadinstitute/hellbender/utils/codecs/GENCODE/GencodeGtfFeature.java

```diff
@@ -575,12 +575,12 @@ public int hashCode() {
     /**
      * Checks if {@code other} is contained within this {@link GencodeGtfFeature}.
      * Comparison is made using {@link SimpleInterval#contains(Locatable)} ala {@link GencodeGtfFeatureBaseData#genomicPosition}
-     * @param other {@link GencodeGtfFeature} of which to check the bounds.
+     * @param other {@link Locatable}-derived class of which to check the bounds.
      * @return true if {@code other} is contained within the bounds of this {@link GencodeGtfFeature}, false otherwise.
      */
-    public boolean contains(final GencodeGtfFeature other) {
+    public <T extends Locatable> boolean contains(final T other) {
```

Comment:
Does this need to be a generic? I think that using `public boolean contains(Locatable other)` might work...

Reply:
You might be right - I'll try some tests with it to see.

Reply:
Yup - I'll pull it back to being a pure Locatable.